<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_bulk_card_import</class>
 <widget class="QDialog" name="frm_bulk_card_import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_select_path_bulkcard_upload">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_import_bulkcard">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset_bulkcard">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_path_bulkcard">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path for Bulk Card Upload</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_path_bulkcard">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lb_warning_bulk_upload_1">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>105</y>
     <width>671</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Warning for correct folder structure and filename syntax!!!</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bulk_card_upload_section">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>601</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bulk Card Import for Eqsl downloaded by AC9HP Downloader</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel_frm_bulk_import">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
